### PR TITLE
Add screenshots to VSCode extension spec

### PR DESCRIPTION
**Pull request by carmenberndt**
Added screenshots showing syntax highlighting, linting, formatting and jump to definition and added the new feature jump-to-definition.

### CODE REVIEW

**Comment by janpio** [comment]
"Linting" doesn't work for me at the rendered version: https://github.com/prisma/specs/blob/updateVSCodeExtensionSpec/vscode-extension/README.md

Can we influence the image size so it shows the font in a similar size than the document itself, and in a consistent size over all images?

File: vscode-extension/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
   - Spec: In Progress 🚧
   - Implementation: Fully implemented ✅
 
-The Prisma VSCode Extension adds syntax highlighting, formatting and linting for `.prisma` files to VSCode.  
+The Prisma VSCode Extension adds syntax highlighting, formatting, linting and Go to Definition for `.prisma` files to VSCode.
```
Comment (janpio) [inline vscode-extension/README.md:8]: Unrelated but possibly nice change:

```suggestion
The Prisma VSCode Extension adds [Syntax Highlighting](#syntax-highlighting), [Linting](#linting), [Formatting](#formatting) and [Go to Definition](#go-to-definition) for `.prisma` files to VSCode.  
```

**Comment by janpio** [comment]
(Updated the PR title to be a bit more descriptive)

**Comment by carmenberndt** [comment]
I uplloaded the images to Imgur already, I'll revert that now.

**Comment by carmenberndt** [comment]
> "Linting" doesn't work for me at the rendered version: https://github.com/prisma/specs/blob/updateVSCodeExtensionSpec/vscode-extension/README.md

Are you sure? When I click on the link everything works fine.
> Can we influence the image size so it shows the font in a similar size than the document itself, and in a consistent size over all images?

I think so, yes, will do that.

**Comment by janpio** [comment]
> Are you sure? When I click on the link everything works fine.

Yes:
image

"Open image in new tab" also shows a 404: https://github.com/prisma/specs/raw/updateVSCodeExtensionSpec/vscode-extension/images/lintingScreenshot.PNG?raw=true

**Comment by carmenberndt** [comment]
Should be fixed now.

File: vscode-extension/README.md [hunk 2]
```diff
@@ -36,22 +38,35 @@ The following elements of a schema file are highlighted:
 
 - Model Blocks (`model`)
 - Config Blocks (`datasource` and `generator`)
-- Enum Blocks (`enum`)
+- Enum Blocks (`enum`) (not supported yet)
```
Comment (janpio) [inline vscode-extension/README.md:41]: Please create an issue for this in the appropriate repo if it does not exist yet, if it does maybe link it from here.

File: vscode-extension/README.md [hunk 3]
```diff
@@ -36,22 +38,35 @@ The following elements of a schema file are highlighted:
...
 - Double and triple comments (`//` and `///`)
 
-TODO appropriate screenshot that shows all functionality
+![Alt text](./images/syntaxHighlighting.png?raw=true "Syntax Highlighting example")
```
Comment (janpio) [inline vscode-extension/README.md:44]: Nitpick: The images should have an appropriate alternative text as well. (applies to all images)

File: vscode-extension/README.md (hunk 2 above)
Comment (carmenberndt) [reply vscode-extension/README.md:41]: should I create a new PR for this, since the branch is already merged?

Comment (carmenberndt) [reply vscode-extension/README.md:41]: I double checked: this is supported, just not for sqlite as sqlite does not support enums. I'll add a screenshot for that

**Pull request merged**